GitHub-347: Clarifying not to bold unlabeled GUI elements (#349)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

[[user-interface-elements]]
== User interface elements

> Use bold text for all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.

Reviewer: ```suggestion
Use bold text for all graphical user interface (GUI) element names, including menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
```
Redundant and gets rid of an extra layer of commas.

Author: Updated. Also made the items plural to keep it simpler and read a little better (I think at least). Let me know what you think!

[[user-interface-elements]]
== User interface elements

> Bold all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Bold the element name if it appears in the GUI, even if it is not clickable.

Reviewer 2: ```suggestion
Use bold text for all graphical user interface (GUI) element names, including, but not limited to, menu, menu item, button, dialog box, and window names. Use bold text for the element name if the name appears in the GUI, even if the element is not clickable.
```

A few minor comments:

1. Bold is an adjective, not a verb. :) 
2. Some of the pronouns were ambiguous -- please check my suggested rewrites, because I might have misinterpreted the original pronoun targets!

Author: I thought about "bold" as a verb - went as far as checking merriam webster :smile: But I couldn't think of a better alternative and figured I'd leave it alone since it was already there. Thanks for making a great suggestion!

> If the element is not labeled in the GUI, refer to it by a generic description and do not bold it. For example, if a search field is not labeled in the GUI, write it as "the search field", not "the *Search* field".

Reviewer 2: ```suggestion
If an element is not labeled in the GUI, refer to the element by a generic description and do not use bold text. For example, if a search field is not labeled in the GUI, write it as "the search field", not "the *Search* field".
```

> If an element is not labeled in the GUI, refer to the element by a generic description and do not use bold text. For example, if a search field is not labeled in the GUI, write it as "the search field", not "the *Search* field".

Reviewer 3: I wonder if adding a note would be useful (I know the PatternFly link has already been provided) to inform a doc user that if they do not know the name of a tooltip item then they should refer to the Components section of the PatternFly docs? The kebab icon comes to mind, but luckily this is not under the *Components* section list, but the term "kebab" is embedded in the Dropdown sub-section. 

https://www.patternfly.org/v4/components/overflow-menu/html
https://www.patternfly.org/v4/components/dropdown/html/#kebab

Reviewer 2: Might be safer not to include such links? The term "kebab icon" should not be used in our docs :)

Reviewer 3: Yes, best not to link to the "dropdown menu" link, but the general "components" link would be useful. We already mention PatternFly, so one could locate the kebab menu if they go digging.

Author: I think I'd prefer not to link to PatternFly here, because the ISG does override some of the guidance there with the names of the elements. So it's probably the ISG that people should refer to more closely for if they don't know what the UI element is called. (And we can't link to the ISG).

> .Example AsciiDoc

Reviewer 4: You might want to move the Example so that it appears after the first paragraph (Use bold text...)

If you want to add an example to the new paragraph, you could use "Click the edit icon." or "Enter the term in the search field."

> .Example AsciiDoc

Reviewer 4: If this example is to show bolding GUI elements, it should be moved to a position before your new section.

Author: Good suggestion, done!
